samples/notebooks/dotnet/8-chatGPT-with-DALL-E-2.ipynb

```diff
@@ -0,0 +1,194 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Set up the environment, importing Semantic Kernel SDK and SkiaSharp, used to display images inline."
```

Comment:
Add a heading and intro paragraph describing the purpose of this notebook. Remember that these notebooks are for instructional purposes, so we need to spell out the details as much as we can. 

Suggestion:
```
# Using ChatGPT with the Semantic Kernel featuring Dall-E-2
This notebook shows how to make use of the new ChatCompletion API from OpenAI, popularized by ChatGPT. This API brings a new ChatML schema which is different from the TextCompletion API that we've featured so far in other notebooks.

<DESCRIBE SCHEMA HERE> 

When deciding between which one to use, know that ChatGPT models (i.e. gpt-3.5-turbo) are more optimized for chat applications and have been fine-tuned for instruction-following and dialogue. As such, for creating semantic skills with the Semantic Kernel, users may still find the TextCompletion model better suited for certain use cases.
```